app/frontend/src/test/hookWrapper.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { QueryClient, QueryClientProvider } from "react-query";
+import { MemoryRouter } from "react-router-dom";
+import AuthProvider from "../features/auth/AuthProvider";
+
+export default function hookWrapper({ children }: { children: any }) {
```

Comment:
```suggestion
export default function hookWrapper({ children }: { children?: React.ReactNode; }) {
```
I found that if we ever have to use the `initialProps` option in `renderHook` **and** explicitly type the props, it needs `children` to be optional (since it uses the React helper `type PropsWithChildren<P> = P & { children?: ReactNode };`) so let's make it that way so we don't get headaches in future.